glean-core/src/upload/directory.rs

```diff
@@ -0,0 +1,342 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Pings directory processing utilities.
+
+use std::cmp::Ordering;
```

Comment:
Was this moved with `git mv`?

Reply:
Yes, unfortunatelly I didn't realise if I did that and then made changes to file and added those changes, git disconsidered the mv. Anyways, I am fixing the commits and will push the new ones in a bit.